mobile/assets/i18n/en-US.json

```diff
@@ -12,6 +12,10 @@
   "album_viewer_appbar_share_leave": "Leave album",
   "album_viewer_appbar_share_remove": "Remove from album",
   "album_viewer_page_share_add_users": "Add users",
+  "asset_list_layout_settings_dynamic_layout_title": "Dynamic layout",
```

Comment:
Got a better name than "Dynamic layout"? I can't think of anything, but maybe someone more creative than I am can...

Reply:
Great Scott! "Creative Layout"! Or Proportional? Adaptive? Smart?